### ISSUE TITLE
Rftools Dimensions Block/Liquid Dimlet Restrictions.

### ISSUE THREAD

**Issue by duck-mon** [NillerMedDild/Enigmatica6#3226]
**Description**
Write a short description of your suggestion here.
Adjust which blocks/liquids can be used for world generation in rftools dimensions to prevent combinations that would lead to server crashing.
**Why would you like this added/changed?**
I've been randomly generating dimensions with digit dimlets to circumvent the inability to craft dimlets due to the issue described in #3178 and I've noticed how easy it is to roll a dimension that could crash a server. For example: terrain that only consist of sand that endlessly fall into the void, terrain made with flowers or ladders that erupt once a block gets updated, liquid dimlets that make lakes and entire oceans of liquids in their flowing state, etc. Although it's funny to see terrain made of just ladders and buttons, I'm often confronted with the issue of immediate server crashing upon login. It's definitely recoverable using CrashUtilities but I could imagine something like this would be troublesome to solo worlds and servers without access to console. 
**Screenshots if applicable.**

**Comment by RobuxShooters** [comment]
I agree. While we're at it, leaves and various redstone components (most notably observers) should definitely be removed from the list of possible blocks to be generated.

**Comment by TemperedFool** [comment]
I've been poking at RFtools Dimensjons, and will take this all into account. Thank you ^_^

**Comment by NillerMedDild** [comment]
Closing due to #3285 :)
